tests/ttlsfuzzer

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash -e
+# Copyright (C) 2024 Jakub Jelen <jjelen@redhat.com>
+# SPDX-License-Identifier: Apache-2.0
+
+source "${TESTSSRCDIR}/helpers.sh"
+
+if [[ ! -d "${TESTSSRCDIR}/../tlsfuzzer/tlsfuzzer" ]]; then
+    title "TLS fuzzer is not available -- skipping"
+    exit 77;
+fi
+
+TMPFILE="${PWD}/tls-fuzzer.$$.tmp"
+PORT=4433
```

Comment:
I assume this value makes the test dependent on running the tests serially ?
Should it be a different port for each of the tokens or for each run ?

Reply:
right. Given that it is configurable, we should be able to select random number in some free range (or fixed by pkcs11 module?). We could actually do the same thing with the tls test.

Reply:
We can use a base number for each token, say 30000, 32000, 34000, and store it to a file, each test reads the file, increment the content by one and uses that port in that run.

Reply:
bonus is if the allocation script checks if the port is available and skips it if not.

Reply:
The hardcoded port numbers for specific tokens sounds probably as the simplest solution. Added.